docs/content/commands.md

```diff
@@ -70,34 +72,37 @@ public class BuildSystem : ComponentSystem
     {
         for(var i = 0; i < data.Length; i++)
         {
-            var requestSender = data.BuilderRequestSender[i];
+            var requestSender = data.BuildWallSender[i];
             var entityId = data.EntityIds[i];
             
-            BuildRequest buildRequest = new BuildRequest
+            Builder.BuildWall.Request request = new Builder.BuildWall.Request
             {
-                Location = new Location(...),
-                Rotation = new Rotation(...)
-            }
+                TargetEntityId = entityId,
+                Payload = new BuildRequest
+                {
+                    Location = new Location(...),
+                    Rotation = new Rotation(...)
+                }
+            };
+
             
-            requestSender.SendBuildWallRequest(entityId, buildRequest);
+            requestSender.RequestsToSend.Add(request);
         }
     }
 }
 ```
 
-This system runs on a client. It injects all SpatialOS entities in the client's view that have the `Builder` SpatialOS component, and sends a `build_wall` command to each SpatialOS entity (which will be received by the managed worker).
+This system is an example of sending command requests. Unity injects all ECS entities which have the `ShouldSendBuildWallCommand`, `Builder.CommandSenders.BuildWall`, and `SpatialEntityId` components into this system before `OnUpdate`. This system iterates over the injected entities and sends a build_wall command request to each of them.
 
-To send a `build_wall` command in a system, you need to inject `CommandRequestSender<Builder>` into the system like any other ECS component.
+To send a `build_wall` command in a system, you need to inject `Builder.CommandSenders.BuildWall` into the system like any other ECS component.
 
 ### Responding to command requests
 
 When a worker instance receives a command request, the command request is represented with reactive ECS components. 
 
-The Unity GDK attaches a `CommandRequests<T>` ECS component to the specified ECS entity (where `T` implements `IIncomingCommandRequest`). `CommandRequests<T>` holds a list of `T`. The Unity GDK cleans it up at the end of the tick.
+The Unity GDK attaches a `ComponentName.CommandRequests.CommandName` ECS component to the specified ECS entity, where `ComponentName` is the SpatialOS component the command is defined in and `CommandName` is the name of the command in schema. `ComponentName.CommandRequests.CommandName` contains a list of type `ComponentName.CommandName.ReceivedRequest`. The Unity GDK cleans it up at the end of the tick.
```

Comment:
This sentence is a bit of a mind-bend.
"The Unity GDK attaches a `ComponentName.CommandRequests.CommandName` ECS component to the specified ECS entity, where `ComponentName` is the SpatialOS component the command is defined in and `CommandName` is the name of the command in schema."

Trying to unpack...maybe split the sentence up with a colon and replace the "and" with a comma:
The Unity GDK attaches a `ComponentName.CommandRequests.CommandName` ECS component to the specified ECS entity: `ComponentName` is the SpatialOS component the command is defined in,  `CommandName` is the name of the command in schema.